Tests/Samples.Tests/Sql.cs

```diff
@@ -37,7 +37,7 @@ public void ManageSqlDatabaseInElasticPoolTest()
             }
         }
 
-        [Fact]
+        [Fact(Skip = "Regions are not accepting creation of new Windows Azure SQL Database servers at this time")]
```

Comment:
which regions are we running into issues?
Does this test run in all regions or only in one?
If its just one region that try changing the region.

Reply:
I tried use east/central/west. it all run into this issue.